### PR TITLE
Some fixes :-)

### PR DESCRIPTION
**Pull request by dupuchba**

### CODE REVIEW

File: src/acme/main.cljd [hunk 1]
```diff
@@ -16,43 +16,42 @@
...
+                  nil))])
 
 (defn- ^w/Widget drawer-builder [context]
   (f/widget
     m/Drawer
-    (w/ListView
-      .padding w.EdgeInsets/zero
-      .children (drawer-children context))))
+    (w/ListView .padding w/EdgeInsets.zero)
+    .children (drawer-children context)))
```
Comment (aangiel) [inline src/acme/main.cljd:54]: To be honest, I prefer my formatting. It lines up properties and indents them as be owned by w/ListView

File: src/acme/main.cljd [hunk 2]
```diff
@@ -91,8 +90,9 @@
      .label "Error"
      .icon (-> context pw/PlatformIcons .-error w/Icon))])
 
-(def ^:private darkDefaultCupertinoTheme
-  (c/CupertinoThemeData .brightness d.Brightness/dark))
+(def ^:private ^c/CupertinoThemeData darkDefaultCupertinoTheme
```
Comment (aangiel) [inline src/acme/main.cljd:93]: Why we need that?

File: src/acme/main.cljd [hunk 3]
```diff
@@ -101,20 +101,20 @@
       (.dark m/ThemeData)
       .cupertinoOverrideTheme
       (c/CupertinoThemeData
-        .brightness d.Brightness/dark
+        .brightness d/Brightness.dark
```
Comment (aangiel) [inline src/acme/main.cljd:104]: Is this only for convention that namespaces is always followed by slash?

**Review by aangiel — COMMENTED** [review]
I'm very proud of myself that the Creator has only few small fixes :D Thanks!

File: src/acme/main.cljd (hunk 2 above)
Comment (aangiel) [reply src/acme/main.cljd:93]: Ok, that's clear after #clojuredart Slack channel message that it removes dynamic warning.

File: src/acme/main.cljd (hunk 1 above)
Comment (dupuchba) [reply src/acme/main.cljd:54]: no problem!

File: src/acme/main.cljd (hunk 3 above)
Comment (dupuchba) [reply src/acme/main.cljd:104]: we support both but it's also simpler to use this form when try to get nested property like so
`m/Colors.grey.shade100`

**Review by aangiel — CHANGES_REQUESTED** [review]
@dupuchba could you rollback commented formatting changes?

**Comment by dupuchba** [comment]
hey @aangiel unfortunately I am very busy today.
The idea of the PR was more to show you alternatives, there is no "truth" when it comes to writing code (although sometimes there is really bad code :D ) . Anyway just take what you want from this PR (or nothing).

**Comment by aangiel** [comment]
@dupuchba Ok. Yeah, that's the point. Every code should be treated individually. Yes, I saw a looot of bad code (which was readable, but unmaintenable). 

You know, it's my first incoming PR :D So I treat it very serious :D 

I'm merging it, because it's very valuable. Thank you a lot!

**Review by aangiel — APPROVED** [review]

**Pull request merged**